test/cpp/lazy/test_lazy_ops.cpp

```diff
@@ -21,7 +21,9 @@ namespace lazy {
 #ifndef FBCODE_CAFFE2
 
 namespace {
-  // This registers the torchscript backend, without which lazy device won't work
+// This registers the torchscript backend, without which lazy device won't work.
+// FIXME: This registers the backend for the whole test binary. We should probably
```

Comment:
We don't currently have a 'de-init' function, do we want to add one?

Does calling init multiple times break anything (should we make that safe, or make it err?)

Reply:
Calling init multiple times break things. PyTorch will complain that the lazy backend get re-registered. However, I don't know if we can add a de-init thing. I try to look for APIs that can un-register the lazy backend but failed...

Reply:
well i guess it would be easier to make our init function stateful, so it only really inits once?

Reply:
I love the idea. A `static bool` should do.